app/models/player.rb

```diff
@@ -1,6 +1,9 @@
 class Player < ApplicationRecord
   include Sortable
 
+  has_many :game_players, -> { order(position: :asc) }, dependent: nil, inverse_of: :game_players
```

Comment:
Could you explain
1. what does `dependent: nil` do?
2. what does `inverse_of` do?

Reply:
`dependent: nil` does nothing (it's a default) I put it how pass the lint. 
`inverse_of` in these case also does nothing, it's a default because Active Record can guess the inverse of the association based on the name of the class. I put it how pass the lint.

Reply:
I don't know about `dependent: nil` tbh. If you delete a player, when you do game.players you will get an `ActiveRecord::RecordNotFound` 🤔

Reply:
I'd say to keep it as dependent destroy